src/main/java/org/swisspush/redisques/util/RedisquesAPI.java

```diff
@@ -174,22 +183,32 @@ public static JsonObject buildGetQueuesCountOperation(){
     }
 
     /**
+     * Evaluate the size of all queues matching the optional given filter pattern.
+     *
      * @param filterPattern
      *      Filter pattern. Method handles {@code null} gracefully.
      */
     public static JsonObject buildGetQueuesCountOperation(String filterPattern) {
-        if (filterPattern != null) {
-            return buildOperation(QueueOperation.getQueuesCount, new JsonObject().put(FILTER, filterPattern));
-        } else {
-            return buildOperation(QueueOperation.getQueuesCount);
-        }
+        return buildOperation(QueueOperation.getQueuesCount, new JsonObject().put(FILTER, filterPattern));
```

Comment:
Is this change by intent? I cannot see any benefit of it.

Reply:
It was by intent. But I reverted it now to be backward compatible even if not really necessary I think.